src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -73,6 +73,10 @@
     </f:entry>
 
     <f:dropdownDescriptorSelector title="${%Yaml merge strategy}" field="yamlMergeStrategy" default="${descriptor.defaultYamlMergeStrategy}" />
+    
+    <f:entry field="inheritYamlMergeStrategy" title="${%Inherit yaml merge strategy}" >
+      <f:checkbox default="false"/>
```

Comment:
Already defaults to false so this is superfluous.
```suggestion
      <f:checkbox/>
```

Reply:
Will do